### PR TITLE
fix(readme): add deis namespace to kubectl commands

### PR DESCRIPTION
**Pull request by rimusz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ $ helm repo add deis https://github.com/deis/charts
 $ helm up
 $ helm fetch deis/deis
 $ helm install deis
```
Comment (bacongobbler) [inline README.md:24]: We also need to add `--namespace=deis` here.

Comment (rimusz) [inline README.md:24]: @bacongobbler this is exactly what this PR fixes

Comment (bacongobbler) [inline README.md:24]: yeah but it's missing on this line as well. The official instructions say to use `helm install deis --namespace=deis` but it's not reflected here, just on the `kubectl` commands below. :)

Comment (rimusz) [inline README.md:24]: why do we need `helm install deis --namespace=deis` if the chart's manifest files have `namespace: deis` and we have `deis-namespace.yaml` too?

Comment (bacongobbler) [inline README.md:24]: it looks like this was changed while I was gone. Disregard

Comment (rimusz) [inline README.md:24]: no worries, things get changed a lot :)

**Pull request merged**